routes/clucksRouter.js

```diff
@@ -0,0 +1,82 @@
+const express = require('express');
+const knex = require('../db/client');
+const router = express.Router();
+
+router.get('/', (request, response) => {
+    knex('clucks').orderBy('created_at', 'desc')
+    .then(clucks => {
+        let hashtags = {};
+        let indexes = [];
+        clucks.forEach(element => {
+            if (element.content.includes('#')) {
+                element.content = element.content.replaceAll('\r\n', ' ');
```

Comment:
`TypeError: element.content.replaceAll is not a function`

Reply:
I don't get TypeError when I run the program through nimbus, could there be a difference with environments?

Reply:
I didn't modify anything in knexfile. If you still doubt this is due to a setup issue and want to claim the marks for trending tags, you can get one of the instructors or TAs to test it out and have them message me that it's working as intended.

Reply:
@alecchanpascal Actually, I can try this again and get back to you by Saturday. May I ask the node version you are using?

Reply:
Thank you and it's v17.6.0